patternlab/styleguide/source/assets/scss/base-theme.scss

```diff
@@ -13,6 +13,9 @@
 @import "06-theme/00-base/fonts";
 @import "06-theme/00-base/pikaday";
 
+// Print styles

```

Comment:
Comments should end in a full-stop period.

Reply:
The MASS coding standards don't require the period at the end of a comment